### PR TITLE
PMM-5813 create files to destinguish AMI from OVF

### PR DESCRIPTION
**Pull request by Sevlin**

### CODE REVIEW

File: ansible/roles/ami-ovf/tasks/main.yml [hunk 1]
```diff
@@ -28,6 +28,22 @@
...
+    - name: OVF                        | Create /.pmm-ovf file
+      when: ansible_virtualization_type == "virtualbox"
+      file:
+        path: /.pmm-ovf
+        owner: root
+        group: root
+        mode: 0600
+
+    - name: AMI                        | Create /.pmm-ami file
+      when: ansible_virtualization_type != "virtualbox"
```
Comment (AlekSi) [inline ansible/roles/ami-ovf/tasks/main.yml:40]: Not VirtualBox

https://github.com/Percona-Lab/percona-images/blob/b1518d0087cef704e5f3549c26df82d5b3f286ed/ansible/pmm/main.yml#L112-L131

File: ansible/roles/ami-ovf/tasks/main.yml [hunk 2]
```diff
@@ -28,6 +28,22 @@
...
+      file:
+        path: /.pmm-ami
+        owner: root
+        group: root
+        mode: 0600
+
     - name: Packages                   | Remove PMM2 Testing Server repository
```
Comment (AlekSi) [inline ansible/roles/ami-ovf/tasks/main.yml:47]: Are you sure that is PMM 1.x file?

Comment (AlekSi) [reply ansible/roles/ami-ovf/tasks/main.yml:47]: I'm pretty sure that file is not used for PMM 1.x builds. /cc @hors

Comment (hors) [reply ansible/roles/ami-ovf/tasks/main.yml:47]: @Sevlin  if you need to add it for PMM 1.x you need to check [pmm/main.yml](https://github.com/Percona-Lab/percona-images/blob/master/ansible/pmm/main.yml)

File: ansible/roles/ami-ovf/tasks/main.yml [hunk 3]
```diff
@@ -28,6 +28,7 @@
       when: ansible_virtualization_type == "virtualbox"
       command: swapon -a
 
+
```
Comment (hors) [inline ansible/roles/ami-ovf/tasks/main.yml:31]: please remove this blank line

Comment (AlekSi) [reply ansible/roles/ami-ovf/tasks/main.yml:31]: ^

Comment (AlekSi) [reply ansible/roles/ami-ovf/tasks/main.yml:31]: ^

**Pull request merged**